ReleaseNotes.md

```diff
@@ -17,55 +15,53 @@ These release notes cover the following topics:
 
 ## **Package Description**
 
-This package is a courtesy build of Pivotal's [https://github.com/appsuite/oss-httpd-build](https://github.com/appsuite/oss-httpd-build) framework, to provide a reference for customers of Pivotal's support for the open source Apache HTTP Server project. This includes Apache HTTP Server (httpd), along with a number of frequently updated library components (dependencies). These are distributed to the general public from the [https://network.pivotal.io/products/p-apache-http-server](https://network.pivotal.io/products/p-apache-http-server) download location.
-
-This package is structured to allow parallel installation of multiple releases of Apache HTTP Server and related components. It contains one directory tree, labeled as httpd-2.4.41-190812 which represents the version of httpd and the effective date of all components bundled in the package, in this case, all of the components current as of 2019 August 12.
+This package consists of Apache HTTP Server (httpd), along with a number of frequently updated library components (dependencies) required by httpd or one of its optional modules. The package was assembled with VMware's [https://github.com/appsuite/oss-httpd-build](https://github.com/appsuite/oss-httpd-build) OSS Build and Server Instance Schema for Apache HTTP Server.

```

Comment:
For consistency, "The package was assembled..." should be "This package is assembled...